style.css

```diff
@@ -108,26 +107,544 @@ a {
+  justify-content: flex-start;
+  gap: 10px;
+  background: #f7f7f9;
+  border-radius: 2px;
+  padding: 5px;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  color: #253858;
+}
+
+.lg2 {
+  display: none;
+}
+
+.frameworks {
+  padding: 10px 10px;
+}
+
+hr {
+  height: 1px;
+  border: none;
+  background: #dfe1e6;
+}
+
+.frameworks,
+.languages,
+.skills {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.skills {
+  padding: 10px 10px;
+}
+
+/* contact me */
+.contact-me {
+  background: #6070ff;
+  border-top-left-radius: 80px;
+  padding-bottom: 20px;
+}
+
+.bg-color-1 {
+  background-color: #fff;
+}
+
+.contact-container {
+  padding: 100px 0 0 0;
+}
+
+.lg-text-4 {
+  color: #fff;
+  font-size: 40px;
+  font-weight: bolder;
+  line-height: 48px;
+  letter-spacing: 0.37px;
+  margin: 0 20px 10px 20px;
+  text-align: center;
+}
+
+.image-container {
+  display: flex;
+  align-items: self-start;
+  justify-content: right;
+}
+
+.bg-image-2 {
+  background-image: url("./assets/contact-form-shapes-mobile.svg");
+  background-position: 80px 5px;
+  background-repeat: no-repeat;
+  background-size: 100% 100%;
+}
+
+.sm-text-3 {
+  font-weight: 300;
+  font-size: 24px;
+  text-align: center;
+  color: #ebebff;
+  margin: 0 20px 40px 20px;
+}
+
+/* form */
+.form {
+  display: flex;
+  margin: 20px;
+  flex-direction: column;
+  justify-content: center;
+  align-items: flex-start;
+}
+
+.form-text {
+  font-size: 28px;
+  text-align: center;
+  margin: 20px;
+}
+
+input {
+  border-top-style: hidden;
+  border-right-style: hidden;
+  border-left-style: hidden;
+  border-bottom-style: solid;
+}
+
+input[type="text"],
+[type="email"] {
+  width: 100%;
+  padding: 16px 14px;
+  margin: 0 0 10px 0;
+  box-sizing: border-box;
+  border-radius: 4px;
+}
+
+textarea {
+  width: 100%;
+  height: 170px;
+  padding: 12px 8px;
+  border: none;
+  border-radius: 2px;
+  background-color: #fbf8f7;
+  margin: 0 0 10px 0;
+}
+
+textarea:hover::placeholder {
+  background: #d1cbc9;
+}
+
+.btn-1 {
+  padding: 12px;
+  border-radius: 8px;
+  font-weight: 500;
+  font-size: 17px;
+  letter-spacing: 0.03em;
+  background-color: #fff;
+  color: #6070ff;
+  margin: 0 0 20px 0;
+}
+
+.btn-1:active {
+  background: rgba(255, 255, 255, 0.849);
+  font-weight: 600;
+  color: #333;
+}
+
+@media screen and (min-width: 786px) {
+  #icon-menu {
+    display: none;
+  }
+
+  .normal-button ul {
+    display: flex;
+    align-items: center;
+    justify-content: space-evenly;
+    visibility: visible;
+  }
+
+  .normal-button li {
+    margin-left: 30px;
+    font-size: 16px;
+    font-weight: 500;
+    color: #344563;
+  }
+
+  .normal-button a {
+    font-size: 1.2rem;
+    color: #344563;
+  }
+
+  .bg-image-1 {
+    background-image: url("./assets/desktop-header.svg");
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: contain;
+    border: 1px solid transparent;
+  }
```

Comment:
- [ ] Kindly make sure that all screen that has `min-width: 786px` (as it is your media query breakpoint) their headline section looks exactly as provided in the Figma :wink: I notice that everything starts to work fine at `min-width: 1055px`. Kindly make sure that at the breakpoint you had set `min-width: 786px` even there everything is working fine :wink: 

   - ### Your design (width: 787px)
   ![image](https://user-images.githubusercontent.com/87197412/186934073-9daabf30-6f0b-4bbd-af47-aba93ca5b84c.png)


  - ### Figma design
  ![](https://user-images.githubusercontent.com/87197412/186931071-7cbb108b-d723-4190-9baf-1aefa7dda3d7.png)

Reply:
Resolved